File: Day-15/Pattern21.js
function PatternTwentyone(n) {
  let i = 1;
  while (i <= n) {
    let str = "";
    let space = i - 1;
    while (space) {
      str += " ";
      space--;
    }
    let star = n - i + 1;
    while (star) {
      str += "*";
      star--;
    }
    i++;
    console.log(str);
  }
}

PatternTwentyone(4);

File: Day-15/Pattern21.py
n = int(input())

def PatternTwentyone(n):
    i = 1
    while(i<=n):
        space = i - 1

        while(space):
            print(' ', end='')
            space-=1

        star = n-i+1
        
        while(star):
            print('*', end='')
            star -= 1
        
        print(end='\n')
        i+=1    
    

PatternTwentyone(n)